src/bundle/ParamConverter/LanguageCodeParamConverter.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\ParamConverter;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\LanguageService;
+use eZ\Publish\API\Repository\Values\Content\Language;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class LanguageCodeParamConverter implements ParamConverterInterface
```

Comment:
Class name is very confusing. Look, you have both `LanguageCodeParamConverter` as well as `LanguageParamConverter`. Also, `LanguageCodeParamConverter` is misleading as I'd expect it to use `languageCode` route parameter to convert from. `TranslationLanguageParamConverter` is better bet but personally I'd avoid using them at all since they sometimes lead to unwanted results.